src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Lotto {
+
+    private List<LottoNumber> lottoNumberList;
+
+    public Lotto(List<LottoNumber> lottoNumberList) {
+        this.lottoNumberList = lottoNumberList;
+    }
+
+
+    public static Lotto of(List<Integer> numbers) {
+        List<LottoNumber> lottoNumberList = new ArrayList<>();
+        for (Integer number : numbers) {
+                lottoNumberList.add(LottoNumber.of(number));
+        }
+        return new Lotto(lottoNumberList);
+    }
+
+
+    @Override
+    public String toString() {
+        return lottoNumberList.stream()
+                .map(LottoNumber::toString)
+                .collect(Collectors.joining(","));
+    }
+
+    public int containsCount(Lotto lastWeekWinningLotto) {
+        int count = 0;
+        for (LottoNumber lottoNumber : lottoNumberList) {
+            count += lastWeekWinningLotto.increaseCount(lottoNumber);
```

Comment:
```suggestion
            count += contains(lottoNumber) ? 1 : 0;
```
과 같은 표현식으로 가독성을 확보해보는 것은 어떨까요?
increaseCount 메소드 내부는 엄밀히얘기하면 count 를 증가시키는게 아니라 증가시킬 count 를 반환하는 메소드인데 현재 네이밍으로는 혼동이 올 수 있을 것 같아요!